src/classes/UTIL_Describe_TEST.cls

```diff
@@ -362,4 +362,52 @@ public with sharing class UTIL_Describe_TEST {
 
         System.assertNotEquals(0, picklistEntriesApiToLabel.size()); 
     }
+
+    /**
+    * @description Test to ensure isLookupFieldTo() returns true when a specified field is of type lookup to the 
+    * specified SObject type.
+    */ 
+    @isTest 
+    private static void isLookupToAccountTrue(){
+
+        SObject conObj = new Contact(LastName = 'testCon');
+        Schema.DescribeFieldResult acctFieldDesc = conObj.getSObjectType().getDescribe().fields.getMap().get('AccountId').getDescribe();
```

Comment:
Within the context of this test, why not use a pre-existing combination of functions to get this information from UTIL_Describe? This is an effective and specific test, so there is value in keeping it isolated in the way you have built it as well.